pages/resources.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+permalink: /resources
+title: Student Resources
+---
+
+Next House has many resources for students to access from the Residential Peer Mentors Program to our Area Director and Heads of House. All the members of the community are there to help and support students and specifically freshmen as they accustom to Next and the Institvte.
+
```

Comment:
In the old, html version, there was a gap here. This version removes that, so I think it looks a bit too cramped now:
![image](https://user-images.githubusercontent.com/8433005/29240583-1e413c8a-7fa3-11e7-9640-1c8adb1759a4.png)
(same comment applies to other md pages)